ui/server.py

```diff
@@ -307,14 +486,21 @@ def getModels():
 
     return models
 
-@app.get('/modifiers.json')
-def read_modifiers():
-    headers = {"Cache-Control": "no-cache, no-store, must-revalidate", "Pragma": "no-cache", "Expires": "0"}
-    return FileResponse(os.path.join(SD_UI_DIR, 'modifiers.json'), headers=headers)
-
-@app.get('/output_dir')
-def read_home_dir():
-    return {outpath}
+@app.get('/get')
```

Comment:
How about `/get/{key}`, so that it follows a RESTful pattern?

Reply:
I used query to make sure that for replies where we don't specify a header, that the result won't be cached.
Mosts browsers will explicitly disable cache for url with a query.
https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/QueryStringParameters.html

Reply:
We call send a cache header fairly easily (see the cache header sent in another app path). For now, sending a "never cache" header is correct, since we're sending live config data like models and app config, which should always be fresh